core/example/longrange/CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ if(Cabana_ENABLE_Cuda)
   find_package(CUDA REQUIRED)
   add_executable(SPME spme_example.cpp)
   target_link_libraries(SPME cabanacore ${CUDA_CUFFT_LIBRARIES})
-elseif()
+else()
```

Comment:
:(
Was it evaluating to false?

Reply:
Yes, apparently CMake treats this as `elseif(FALSE)`. This is why the build tests passed - it never built SPME.

It looks like similar issues were encountered by others, and even within CMake source itself? [https://gitlab.kitware.com/cmake/cmake/merge_requests/3515](url)

Reply:
Haha.  Thanks for sharing the reference!